docs/search-ui-react.facetsprops.onlyrenderchildren.md

```diff
@@ -0,0 +1,13 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@yext/search-ui-react](./search-ui-react.md) &gt; [FacetsProps](./search-ui-react.facetsprops.md) &gt; [onlyRenderChildren](./search-ui-react.facetsprops.onlyrenderchildren.md)
+
+## FacetsProps.onlyRenderChildren property
+
+If set to true, only the facets specified in the children are rendered. If set to false, all facets are rendered with the ones specified in the children overrode. Default to false.
+

```

Comment:
I think it should be `overridden` here, based on [this](https://english.stackexchange.com/questions/75786/overrode-vs-overridden)

Reply:
Done